addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -227,6 +235,22 @@ export class LinkPopover extends Component {
         }
     }
 
+    openAdvanceOption() {
```

Comment:
```suggestion
    toggleAdvanceOption() {
```

You can rename the method and reuse instead `previousPopup`, so there won't be need of extra method.